src/main/java/org/openstreetmap/atlas/utilities/vectortiles/MinimumZoom.java

```diff
@@ -0,0 +1,166 @@
+package org.openstreetmap.atlas.utilities.vectortiles;
+
+/**
+ * This utility class is where you set the minimum zoom for features based on their tags. The values
+ * you see here for various min zooms is inspired for what you see in the standard OpenStreetMap
+ * carto style. This is very loosely based off of the minimum zooms you see for various types of
+ * features. Note that there is definitely more work that needs to be done to refine our min zooms.
+ * https://github.com/gravitystorm/openstreetmap-carto
+ *
+ * @author hallahan
+ */
+final class MinimumZoom
+{
+    private static final int THREE = 3;
+    private static final int FIVE = 5;
+    private static final int SIX = 6;
+    private static final int SEVEN = 7;
+    private static final int EIGHT = 8;
+    private static final int NINE = 9;
+    private static final int TEN = 10;
+    private static final int TWELVE = 12;
+    private static final int THIRTEEN = 13;
+    private static final int FOURTEEN = 14;
+
+    private MinimumZoom()
+    {
+        // Util
+    }
+
+    static int amenity(final String tagValue)
+    {
+        switch (tagValue)
+        {
+            case "hospital":
+                return TWELVE;
+            default:
+                return THIRTEEN;
+        }
+    }
+
+    static int building(final String tagValue)
+    {
+        return THIRTEEN;
+    }
+
+    static int highway(final String tagValue)
```

Comment:
Will this change in the future? It almost makes sense to read this from a configuration file (with this implementation being the default) in case users want to override it..

Reply:
Yeah, I got started making a JSON resource file which I would then parse, but I found that this approach was nice having the switch with fallthroughts. I think you're right that this should be explored. What do you think about leaving it this way for this PR and then having that be part of the next sprint? Or would you rather have me re-do this now?

Reply:
That sounds like a fine plan, maybe create a github issue to track this for some time in the future (next sprint or later).